lib/dataContract/enrichDataContractWithBaseSchema.js

```diff
@@ -18,9 +18,7 @@ function enrichDataContractWithBaseSchema(
   schemaIdPrefix = '',
   excludeProperties = [],
 ) {
-  const rawDataContract = (dataContract instanceof DataContract)
-    ? dataContract.toJSON()
-    : dataContract;
+  const rawDataContract = dataContract.toJSON();
```

Comment:
jsonDataContract

Reply:
No, because we must validate it against our schema